sensors/VNIR/sensor_fixed_metadata.json

```diff
@@ -1,21 +1,23 @@
 {
-  "sensor manufacturer": "Headwall Scientific",
-  "sensor product name": "VNIR",
-  "sensor serial number": "G4-384",
-  "sensor description": "hyperspectral camera to measure visible near infrared (VNIR) radiation",
-  "sensor purpose": "measures spectral reflectance from 380nm to 1000nm",
-  "location in gantry system": "camera box, facing ground",
-  "location in camera box x [m]": "0.877",
-  "location in camera box y [m]": "2.325",
-  "location in camera box z [m]": "0.635",
-  "field of view y [degrees]": "21.0",
-  "optics manufactuer": "Schneider Kreuznach",
-  "optics name": "Xenoplan 1.4/23-0902 CM 120 BK 15 Compac 21-100 1917 32326",
-  "optics focal length [mm]": "22.5",
-  "optics focus setting": "2m",
-  "optics apperture setting": "f/2",
-  "output data format": "hypercube with preview image and additional information provided by sensor",
-  "max scan resolution width [pixel]": "1600",
-  "calibration available": "calibration white/dark reference need to be done constantly during measurements",
-  "sensor id": "vnir camera box"
+  "sensor_manufacturer": "Headwall Scientific",
+  "sensor_product_name": "VNIR",
+  "sensor_serial_number": "G4-384",
+  "sensor_description": "hyperspectral camera to measure visible near infrared (VNIR) radiation",
+  "sensor_purpose": "measures spectral reflectance from 380nm to 1000nm",
+  "location_in_gantry_system": "camera box, facing ground",
+  "location_in_camera_box_m": {
+	"x": "0.877",
+    "y": "2.325",
+    "z": "0.635"
+  },
+  "field_of_view_y_degrees": "21.0",
```

Comment:
i think for consistency this could be 'field_of_view_degrees': {'y': 21.0} even though I don't see an X equivalent